src/components/ProfileInformation/ProfileInformation.types.ts

```diff
@@ -0,0 +1,11 @@
+import { Profile } from 'decentraland-dapps/dist/modules/profile/types'
+
+export type Props = {
+  isLoggedInProfile: boolean
+  profile?: Profile
+  profileAddress?: string
```

Comment:
Shouldn't the profile address be not optional?
```suggestion
  profileAddress: string
```